media-fonts/ibm_plex/ibm_plex-0.5.3.recipe

```diff
@@ -0,0 +1,34 @@
+SUMMARY="The package of IBM’s new typeface, IBM Plex"
+DESCRIPTION="Thanks for your interest in IBM Plex, IBM’s new corporate type family. \
+We welcome your feedback and contributions to this project. \
+IBM Plex is open source, and should be used by IBMers for all typographical situations, \
+replacing Helvetica Neue, whenever possible."
```

Comment:
@alaviss, @dcatt-haiku, what do you think of:
```
DESCRIPTION="IBM Plex is IBM's new corporate type family. It is an open source \
typeface superfamily developed by Mike Abbink for IBM to use in branding. It \
was commissioned to substitute Helvetica Neue and has three typefaces:
* IBM Plex Serif – A serif typeface.
* IBM Plex Sans – A sans-serif typeface.
* IBM Plex Mono – A monospaced typeface."
```

Shall we also remove the lines with `REQUIRES` and `BUILD_REQUIRES` and only keep `BUILD_PREREQUIRES`?
```
BUILD_PREREQUIRES="
	coreutils
	"
```

Reply:
Just to be advised, I took a recommendation from Humdinger to use the Droid (font) recipe as a sort of template, which lead to probably some of the warts you're seeing.  I'll read though all of the comments above and see if I can get it near perfect.

Reply:
Just since there is a lot of back-n-forth above... what changes really need to happen beyond the DESCRIPTION?

Reply:
I'd say that the only changes needed are those in my [last review](https://github.com/haikuports/haikuports/pull/2088#discussion_r160357044): drop  `REQUIRES` and `BUILD_REQUIRES`, change `DESCRIPTION`, and change the style of `BUILD_PREREQUIRES` to have:
```
BUILD_PREREQUIRES="
	coreutils
	"
```

Reply:
@humdingerb, would you be OK with the removal, on this recipe, of the `REQUIRES` and `BUILD_REQUIRES` variables (as they are both set to the empty string), and with a style change for `BUILD_PREREQUIRES`, as shown above?
Thanks!

Reply:
Sure. I didn't know how empty variables are  supposed to be dealt with. By this time, i guess we have quite a mixture... :)

@dcatt-haiku : fbrossons latest suggestion sounds very good.  I'd just list the typefaces, e.g. "IBM Plex Serif" and remove " – A serif typeface" as that's pretty obvious.

Reply:
Thanks @humdingerb!

Regarding the removal of the `"A serif typeface"`, `"A sans-serif typeface"` and `"A monospaced typeface"`, I think we can/should keep them. It's not that obvious to everyone. Novice users don't know that "sans" means "sans-serif" and that "mono" means "monospaced". And Haiku is for everyone, isn't it?

Reply:
I doubt that, but I don't really mind either.

Reply:
Do you doubt that Haiku is for everyone?
Sorry, I'm just kidding :)

Reply:
@dcatt-haiku, if you agree, I can update this PR.

Reply:
@fbrosson sorry that I haven't been able to work on this (work has been busy this week)... I'm fine with you updating the PR.